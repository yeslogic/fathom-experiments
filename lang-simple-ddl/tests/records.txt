type Point {
  x : Int64;
  y : Int64;
}

def test-point-1 : Point := {
  x := 1;
  y := 2;
};

def test-point-2 :=
  { x := 1; y := 2 } : Point;

type A-long-record {
  foo : Int64;
  bar : Int64;
  baz : Int64;
  bananas : Int64;
}

def test-long-record : A-long-record := {
  foo := 1000000;
  bar := 1000000;
  baz := 1000000;
  bananas := 1000000;
};
