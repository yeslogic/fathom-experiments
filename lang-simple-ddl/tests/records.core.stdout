type A-long-record {
  bananas : #Int;
  bar : #Int;
  baz : #Int;
  foo : #Int;
}

type Point {
  x : #Int;
  y : #Int;
}

def test-long-record : A-long-record =
  A-long-record {
    bananas := 1000000;
    bar := 1000000;
    baz := 1000000;
    foo := 1000000
  };

def test-point-1 : Point = Point { x := 1; y := 2 };

def test-point-2 : Point = Point { x := 1; y := 2 };
